spec/lib/jhove_service_spec.rb

```diff
@@ -21,31 +25,40 @@
     expect(@jhove_service.target_pathname).to eq @fixtures.join('temp')
   end
 
-  specify "JhoveService#get_jhove_command" do
+  it "produces the correct get_jhove_command" do
     jhove_cmd = @jhove_service.get_jhove_command(@content_dir)
-    expect(jhove_cmd).to eq "#{@bin.join('jhoveToolkit.sh')} -h xml -o #{@temp.join('jhove_output.xml')} #{@fixtures.join('test_files')}"
-    jhove_cmd = @jhove_service.get_jhove_command(@content_dir, "/my/fileset.txt")
-    expect(jhove_cmd).to eq "#{@bin.join('jhoveToolkit.sh')} -h xml -o #{@temp.join('jhove_output.xml')} edu.stanford.sulair.jhove.JhoveFileset #{@fixtures.join('test_files')} /my/fileset.txt"
+    expect(jhove_cmd).to eq "#{@bin.join('jhoveToolkit.sh')} -h xml -o \"#{@temp.join('jhove_output.xml')}\" \\\"#{@fixtures.join('test_files')}"
```

Comment:
@peetucket confirmed that the lack of close-quote in the expected string is intentional, and done per the jhove documentation.